src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+@use 'sass:list';
+@import './variables';
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.section {
+  padding: 10px;
+  max-width: calc(($width * 7) + 6px);
+}
+
+.calendar {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  $days: mon, tue, wed, thu, fri, sat, sun;
```

Comment:
It's a good practice to define variables at the start of a file or in a separate settings file. This helps with readability and maintainability of the code.